packages/react/src/components/Accordion/AccordionHeading.tsx

```diff
@@ -1,73 +1,32 @@
-import { ChevronDownIcon } from '@navikt/aksel-icons';
 import cl from 'clsx/lite';
-import type { HTMLAttributes, MouseEventHandler, ReactNode } from 'react';
-import { forwardRef, useContext } from 'react';
+import type { HTMLAttributes, ReactNode } from 'react';
+import { forwardRef } from 'react';
 
-import { Heading, Paragraph } from '../Typography';
-
-import { AccordionItemContext } from './AccordionItem';
+import { Paragraph } from '..';
 
 export type AccordionHeadingProps = {
-  /**
-   * Heading level. Use this to make sure the heading is correct according to you page heading levels
-   * @default 1
-   */
-  level?: 1 | 2 | 3 | 4 | 5 | 6;
-  /** Handle when clicked on header */
-  onHeaderClick?: MouseEventHandler<HTMLButtonElement> | undefined;
   /** Heading text */
   children: ReactNode;
-} & HTMLAttributes<HTMLHeadingElement>;
+} & HTMLAttributes<HTMLElement>;
 
 /**
- * Accordion heading component, contains a button to toggle the content.
+ * Accordion header component, contains a button to toggle the content.
  * @example
- * <AccordionHeading>Header</AccordionHeading>
+ * <AccordionHeader>Header</AccordionHeader>
  */
-export const AccordionHeading = forwardRef<
-  HTMLHeadingElement,
-  AccordionHeadingProps
->(({ level = 1, children, className, onHeaderClick, ...rest }, ref) => {
-  const context = useContext(AccordionItemContext);
-
-  if (context === null) {
-    console.error(
-      '<Accordion.Heading> has to be used within an <Accordion.Item>',
-    );
-    return null;
-  }
-
-  const handleClick: MouseEventHandler<HTMLButtonElement> = (e) => {
-    context.toggleOpen();
-    onHeaderClick?.(e);
-  };
-
-  return (
-    <Heading
-      ref={ref}
-      size='xs'
-      level={level}
-      className={cl('ds-accordion__heading', className)}
-      {...rest}
-    >
-      <button
-        type='button'
-        className={cl('ds-accordion__button', `ds-focus`)}
-        onClick={handleClick}
-        aria-expanded={context.open}
-        aria-controls={context.contentId}
-      >
-        <ChevronDownIcon
-          aria-hidden
-          className='ds-accordion__expand-icon'
-          fontSize={'1.5rem'}
+export const AccordionHeading = forwardRef<HTMLElement, AccordionHeadingProps>(
+  function AccordionHeading({ className, ...rest }, ref) {
+    return (
+      <Paragraph asChild size='sm'>
+        <u-summary
+          class={cl(
+            'ds-paragraph ds-paragraph--sm ds-line-height--md', // Very TMP fix awaiting CSS modules
```

Comment:
Does `asChild` work for the web-component (since you repeated the paragraphs classes here)?